x-pack/filebeat/module/squid/log/test/access2.log-expected.json

```diff
@@ -0,0 +1,5610 @@
+[
+    {
+        "@timestamp": "2002-10-23T10:25:29.000Z",
+        "event.action": "TCP_MISS",
+        "event.code": "GET",
+        "event.dataset": "squid.log",
+        "event.module": "squid",
+        "event.original": "1035368729.430    371 210.8.79.228 TCP_MISS/200 2136 GET http://www.fas.harvard.edu/~hpcws/nav/navbar_r3_c6.jpg - FIRST_PARENT_MISS/proxy2.syd.connect.com.au image/jpeg",
+        "fileset.name": "log",
+        "http.response.body.content": "navbar_r3_c6.jpg",
+        "input.type": "log",
+        "log.offset": 0,
+        "observer.product": "Proxy",
+        "observer.type": "Proxies",
+        "observer.vendor": "Squid",
+        "related.ip": [
+            "210.8.79.228"
+        ],
+        "related.user": [
+            "-"
```

Comment:
I'm more inclined to leave this one as is for now, until a mechanism is added to make this kind of conditionals per device instead of adding code to all the pipelines for this specific case